agent/api/task_unix.go

```diff
@@ -37,6 +37,8 @@ const (
 	maxTaskVCPULimit = 10
 	// Reference: http://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_ContainerDefinition.html
 	minimumCPUShare = 2
+
+	BytesPerMegabyte = 1024 * 1024
```

Comment:
Could this `const` be made local to this package?

Reply:
done!